content/syllabuses/data-eng.md

```diff
@@ -20,13 +20,13 @@ _db_id: 4
 - {{% contentlink path="projects/first-personal-website" %}}
 - {{% contentlink path="topics/how-to-ask-for-help-with-your-code" %}}
 - {{% contentlink path="language-agnostic/intro-to-hackerrank" flavour="python"%}}
-- {{% contentlink path="workshops/big-picture-web-dev" %}}
+- {{% contentlink path="topics/big-picture-web-dev" %}}
 - NCIT 10: Workshop: Giving and receiving constructive feedback
 - NCIT 06: Web design for business
 - NCIT 07: Building an online business
 - NCIT 12: Business and technology
 - NCIT 09: Ethics and Professionalism in IT
-- {{% contentlink path="workshops/general-coding-best-practices" %}}
+- {{% contentlink path="topics/general-coding-best-practices" %}}
```

Comment:
Is content/topics/clean-code/general/_index.md not already in the syllabus? Does it need to be upgraded to add any missing content covered by the workshop?

Best thing to do is upgrade the topic and depreciate the workshop instead of having them go through both.